templates/new/config/config_umbrella.exs

```diff
@@ -0,0 +1,25 @@
+# This file is responsible for configuring your application and its
+# dependencies.
+#
+
+if Mix.target() != :host do
+  # Enable the Nerves integration with Mix
+  Application.start(:nerves_bootstrap)
```

Comment:
This is the part that scares me - We shim a lot of tooling in here, even when as the host.